packages/fxa-payments-server/src/routes/Product/SubscriptionUpgrade/index.tsx

```diff
@@ -131,18 +131,18 @@ export const SubscriptionUpgrade = ({
       <Header {...{ profile }} />
       <div className="main-content">
         <div
-          className="product-payment product-upgrade rounded-b-lg rounded-t-none"
+          className="page-title-container"
+          data-testid="subscription-update-heading"
+        >
+          <Localized id="product-plan-change-heading">
+            <h2 className="page-title">Review your change</h2>
+          </Localized>
+          <p className="subheading"></p>
```

Comment:
I meant to ask in my PR, is this p tag even being used for anything?

Reply:
Not that I can tell. I've gone ahead and removed it.